src/env/uat/terraform.tfvars

```diff
@@ -28,6 +28,8 @@ cidr_subnet_pagopa_proxy = ["10.1.240.24/29"]
 cidr_subnet_buyerbanks   = ["10.1.240.32/29"]
 
 cidr_subnet_reporting_fdr = ["10.1.240.40/29"]
+cidr_subnet_gpd           = ["10.1.240.48/29"]
```

Comment:
why different subnet ranges in dev, uat, prod?

Reply:
we do it in to dedicated PR

Reply:
aligned